os/include/netinet/arp.h

```diff
@@ -58,7 +58,7 @@
  ****************************************************************************/
 
 #include <tinyara/config.h>
-
+#include <net/if.h>
```

Comment:
Why include the header file here, if the rest of the file does not use it? Isn't it better to include files where they are used directly?

Reply:
After adding Werror flag we were getting 'arp_pa', 'arp_ha', 'arp_netmask' and 'IFNAMSIZ' related error to overcome that error we have added this header file.